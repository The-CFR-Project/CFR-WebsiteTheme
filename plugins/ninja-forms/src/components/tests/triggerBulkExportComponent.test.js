import { render, cleanup } from '@testing-library/react';
import { globalParams, bulkExportParams } from './mockData';
import { TriggerBulkExportComponent } from '../';

describe( 'submissions-trigger-bulk-export-component', () => {
	afterEach( cleanup );
	it( 'Matches snapshot', () => {
		const props = { globalParams, bulkExportParams };

		const { container } = render(
			<TriggerBulkExportComponent props={ props } />
		);
		expect( container ).toMatchSnapshot();
	} );
} );
