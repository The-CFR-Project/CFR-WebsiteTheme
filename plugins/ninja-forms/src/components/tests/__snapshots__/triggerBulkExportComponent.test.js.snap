// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-trigger-bulk-export-component Matches snapshot 1`] = `
<div>
  <button
    class="components-button is-secondary"
    style="height: 30px; border-radius: 3px;"
    type="button"
  >
    Download Submissions Accross Forms by Date
  </button>
</div>
`;
